src/Common/ItechArt.Common/Logger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+
+namespace ItechArt.Common
+{
+    public class Logger<T> : ILogger<T>
+    {
+        private readonly Microsoft.Extensions.Logging.ILogger<T> _logger;
+
+
+        public Logger()
+        {
+            _logger = new Microsoft.Extensions.Logging.Logger<T>(new LoggerFactory());
+        }
+
+
+        public void Log(LogLevel logLevel, string message, Exception exception = null)
```

Comment:
add other 'helper' methods that will make client to pass all necessary data to log method